packages/components/pages/f-checkout/src/services/addressService/index.js

```diff
@@ -0,0 +1,176 @@
+import uk from './tenants/uk';
+import au from './tenants/au';
+import nz from './tenants/nz';
+import it from './tenants/it';
+
+const tenants = {
+    uk,
+    au,
+    nz,
+    it
+};
+
+const storedLocationKey = 'je-full-address-details';
+
+function toFormattedPostcode (postcode, tenant) {
+    const shouldFormatPostCode = tenant === 'uk';
+
+    if (!shouldFormatPostCode) {
+        return postcode;
+    }
+
+    if (!postcode) {
```

Comment:
this being after the above could mean that you return an `undefined` postcode if shouldFormatPostCode is false, is this intended?

Reply:
It wasn't intentional, thanks!